server/events.ts

```diff
@@ -0,0 +1,22 @@
+import {TypedEmitter} from "tiny-typed-emitter";
+import Client from "./client";
+import {NetworkWithIrcFramework} from "./models/network";
+import {MessageData} from "./plugins/irc-events/message";
+
+export interface ClientEvents {
+	message: (
+		irc: NetworkWithIrcFramework["irc"],
+		network: NetworkWithIrcFramework,
+		client: Client,
+		data: MessageData
+	) => void;
+	notification: (
+		irc: NetworkWithIrcFramework["irc"],
+		network: NetworkWithIrcFramework,
+		client: Client,
+		data: MessageData,
+		notification: any
```

Comment:
Can we type Notification? Is it the same as `notificationData` in irc-events/message.ts?

Reply:
Done